### PR TITLE
Replace PortalGlobals with QuerySession

### PR DESCRIPTION
**Pull request by adamabeshouse**

### CODE REVIEW

**Comment by jjgao** [comment]
@yichaoS could you please review this?

**Comment by jjgao** [comment]
@pieterlukasse this is the PR I was referring.

**Comment by jjgao** [comment]
@adamabeshouse could you do a rebase and solve the conflicts with the current rc branch?

**Comment by yichaoS** [comment]
Looks good to me. Suggestion: maybe we can document how to use window.QuerySession in wiki? I noticed some function you have to use with a callback. That can be confusing sometimes.

**Comment by adamabeshouse** [comment]
@yichaoS done, how's it now?

File: portal/src/main/webapp/js/src/oncoprint/oncoprint-analysis-setup.js [hunk 1]
```diff
@@ -7,23 +7,23 @@ $(document).ready(function() {
...
 
 	
-	window.QuerySession.getGenomicEventData().then(function(response) {
+	$.when(window.QuerySession.getGenomicEventData(), window.QuerySession.getPatientSampleIdMap()).then(function(genomic_event_data, sample_patient_map) {
 			(function invokeOldDataManagers() {
 				var genes = window.QuerySession.getQueryGenes();
-				window.PortalGlobals.setGeneData(response)
-				window.PortalDataColl.setOncoprintData(utils.process_data(response, genes));
-				PortalDataColl.setOncoprintStat(utils.alteration_info(response));
+				window.PortalGlobals.setGeneData(genomic_event_data)
```
Comment (pieterlukasse) [inline portal/src/main/webapp/js/src/oncoprint/oncoprint-analysis-setup.js]: @adamabeshouse : this line could be removed (?)

**Comment by jjgao** [comment]
@adamabeshouse @yichaoS is this ready to be merged?

**Comment by adamabeshouse** [comment]
@jjgao yes, I'll leave removing PortalDataColl to another pull request.

**Pull request merged**